mods/cnc/languages/rules/en.ftl

```diff
@@ -203,59 +210,74 @@ actor-ftnk-husk-name = Flame Tank (Destroyed)
 actor-arty-husk-name = Artillery (Destroyed)
 actor-bggy-husk-name = Nod Buggy (Destroyed)
 actor-bike-husk-name = Recon Bike (Destroyed)
-actor-jeep-husk-name = Hum-Vee (Destroyed)
+actor-jeep-husk-name = Humvee (Destroyed)
 actor-ltnk-husk-name = Light Tank (Destroyed)
 actor-mtnk-husk-name = Medium Tank (Destroyed)
 actor-htnk-husk-name = Mammoth Tank (Destroyed)
 actor-msam-husk-name = Rocket Launcher (Destroyed)
-actor-mlrs-husk-name = Mobile S.A.M. (Destroyed)
+actor-mlrs-husk-name = Mobile SAM (Destroyed)
 actor-stnk-husk-name = Stealth Tank (Destroyed)
 actor-truck-husk-name = Supply Truck (Destroyed)
 
 ## infantry.yaml
 actor-e1 =
    .name = Minigunner
-   .description = General-purpose infantry.
-      Strong vs Infantry
-      Weak vs Vehicles
+   .description = General-purpose infantry
+      Strong vs. Infantry
+      Weak vs. Vehicles
+   .encyclopedia = Costing 100, the humble minigunner is a highly specialized infantry unit meant to soak damage and kill other infantry. Like most TD units, minigunners can see farther than they can shoot, and work well in packs to eliminate enemy infantry.
```

Comment:
With the various encyclopedia entries, I'd prefer to avoid refering to specific numbers such as "Costing 100". It would be very easy for this to get out-of-sync with the game rules, e.g. if somebody decided to tweak the cost to 105 for balance reasons.

All the text suffers the same problem in some sense, e.g. if "minigunners can see farther than they can shoot" was made no longer true. However, specific numbers are much more likely than the more general text to be changed and become incorrect. And it's easy to miss these encyclopedia entries when tweaking the numbers so not having them in the first place would be my preference.